Vizzuality/marxan-cloud
# feat(front): add BLM screenshots to calibration results[MARXAN-1416]

## Add BLM screenshots to calibration results

### Testing instructions

1. Set NEXT_PUBLIC_URL=https://marxan-git-feature-screenshot-map-vizzuality1.vercel.app in your local env.
2. Generate a new calibration and click on one of the BLM values ​​proposed in the chart. 
3. The image should appear on the far right of the chart and should correspond to that chosen BLM value.

**Important:** The same image will be displayed for all values ​​because the correct image is not yet working in marxan49 (update is missing).

### Feature relevant tickets

[MARXAN-1416](https://vizzuality.atlassian.net/browse/MARXAN-1416)

---

## Checklist before submitting

- [ ] Please uncomment NEXT_PUBLIC_URL on `saveScenarioCalibrationRange` baseUrl before merging in develop.
